[2025-04-01T03:40:27.332+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T03:40:27.348+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet manual__2025-04-01T03:40:26.557244+00:00 [queued]>
[2025-04-01T03:40:27.356+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet manual__2025-04-01T03:40:26.557244+00:00 [queued]>
[2025-04-01T03:40:27.357+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-01T03:40:27.369+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): greet> on 2025-04-01 03:40:26.557244+00:00
[2025-04-01T03:40:27.377+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=24380) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-01T03:40:27.376+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v01', 'greet', 'manual__2025-04-01T03:40:26.557244+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmp2392d9rm']
[2025-04-01T03:40:27.378+0000] {standard_task_runner.py:72} INFO - Started process 24381 to run task
[2025-04-01T03:40:27.378+0000] {standard_task_runner.py:105} INFO - Job 113: Subtask greet
[2025-04-01T03:40:27.425+0000] {task_command.py:467} INFO - Running <TaskInstance: our_dag_with_python_operator_v01.greet manual__2025-04-01T03:40:26.557244+00:00 [running]> on host 9663692256f6
[2025-04-01T03:40:27.509+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rajesh' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v01' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T03:40:26.557244+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-01T03:40:26.557244+00:00'
[2025-04-01T03:40:27.510+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T03:40:27.524+0000] {logging_mixin.py:190} INFO - some dict:  {'a': 1, 'b': 2}
[2025-04-01T03:40:27.547+0000] {logging_mixin.py:190} INFO - Hello World! My name is None None, and I am None years old!
[2025-04-01T03:40:27.547+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-01T03:40:27.555+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T03:40:27.555+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v01, task_id=greet, run_id=manual__2025-04-01T03:40:26.557244+00:00, execution_date=20250401T034026, start_date=20250401T034027, end_date=20250401T034027
[2025-04-01T03:40:27.605+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T03:40:27.623+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-01T03:40:27.624+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
